commands/memory.md

```
@@ -0,0 +1,3 @@
+<!-- Created by metaswarm setup. Routes to the metaswarm plugin. Safe to delete if you uninstall metaswarm. -->
+
+Invoke the `/metaswarm:memory` skill to handle this request. Pass along any arguments the user provided.
```

markdownlint-cli2 (0.22.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)